pkg/master/master.go

```diff
@@ -371,6 +374,7 @@ func DefaultAPIResourceConfigSource() *serverstorage.ResourceConfig {
 		policyapiv1beta1.SchemeGroupVersion,
 		rbacv1beta1.SchemeGroupVersion,
 		rbacapi.SchemeGroupVersion,
+		settingsapi.SchemeGroupVersion,
```

Comment:
Was there any discussion on this? We shouldn't be turning alpha APIs on by default.

@jessfraz @pwittrock